### PR TITLE
Add bundle-audit checking

### PR DESCRIPTION
**Pull request by chrismytton**
Use bundler-audit (https://github.com/rubysec/bundler-audit) to report if any of the gems we're using have known vulnerabilities.

This is integrated into Travis, with auto-updating of the vulnerabilities file.

Similar change to https://github.com/everypolitician/viewer-sinatra/pull/15176
Part of https://github.com/everypolitician/everypolitician/issues/493

### CODE REVIEW

**Comment by chrismytton** [comment]
@tmtmtmtm :eyes:

File: Gemfile.lock [hunk 1]
```diff
@@ -1,12 +1,12 @@
 GIT
-  remote: git://github.com/everypolitician/everypolitician-ruby.git
+  remote: https://github.com/everypolitician/everypolitician-ruby.git
```
Comment (tmtmtmtm) [inline Gemfile.lock]: I suspect we'd be better explicitly adding a version to the Gemfile for those cases…

Comment (chrismytton) [inline Gemfile.lock]: I've pushed some new commits, one to remove the unused `everypoliticianbot` gem (d60424d5d375ae8e993ec9bf7e186f07d5e996b0) and another to pin the version of `everypolitician-ruby` to the version we are using here (cb62ad6e6601a166ad6faa28285ab2d3621da8cf).

Comment (chrismytton) [inline Gemfile.lock]: I've pushed some new commits, one to remove the unused `everypoliticianbot` gem (d60424d5d375ae8e993ec9bf7e186f07d5e996b0) and another to pin the version of `everypolitician-ruby` to the version we are using here (cb62ad6e6601a166ad6faa28285ab2d3621da8cf).

Comment (chrismytton) [inline Gemfile.lock]: So this change to `Gemfile.lock` was now generated by bundler instead of by me :)

**Pull request merged**